parsers/NI.py

```diff
@@ -92,14 +90,13 @@ def extract_text(full_text: str, start_text: str, end_text: str | None = None):
         return full_text[start:end]
 
 
-def get_time_from_system_map(text: str):
-    # date format is: "'Actualizado: 07/07/2017 01:00:50 PM'"
-
-    datetime_text = extract_text(text, "Actualizado: ", "'")
-    datetime_arrow = arrow.get(datetime_text, "DD/MM/YYYY hh:mm:ss A")
-    datetime_datetime = arrow.get(datetime_arrow.datetime, TIMEZONE).datetime
-
-    return datetime_datetime
+def get_time_from_system_map(text: str) -> datetime:
+    # date format is: "'InformaciÃ³n en Tiempo Real al 02/04/2024 05:57:40 AM'"
+    datetime_text = extract_text(text, "en Tiempo Real al ", "'")
```

Comment:
Why did this datetime_text change?

Reply:
They seem to have changed the subtitle on their webpage sometime between  [2023/06/02](https://web.archive.org/web/20230206194327/https://www.cndc.org.ni/graficos/MapaSIN/index.php) and [2023/11/28](https://web.archive.org/web/20231128101700/https://www.cndc.org.ni/graficos/MapaSIN/index.php)  👀

Reply:
Man arrow is filled with crazy unexpected stuff!

I'm glad we are removing it!